src/components/Pools/RemoveFarm/index.tsx

```diff
@@ -181,25 +184,35 @@ const RemoveFarm = ({ stakingInfo, version, onClose, onLoading, onComplete, redi
                     <StatWrapper>
                       <Stat
                         title={t('earn.depositedToken', { symbol: 'PGL' })}
-                        stat={stakingInfo?.stakedAmount?.toSignificant(4)}
+                        stat={numeral(
+                          stakedAmount?.toFixed(
+                            stakedAmount?.greaterThan('0') && stakedAmount?.token?.decimals > 1 ? 2 : undefined,
+                          ),
+                        ).format('0.00a')}
                         titlePosition="top"
                         titleFontSize={12}
                         statFontSize={[20, 18]}
                         titleColor="text1"
                         statAlign="center"
+                        toolTipText={stakedAmount?.toExact()}
                       />
                     </StatWrapper>
                   )}
                   {earnedAmount && (
                     <StatWrapper>
                       <Stat
                         title={t('earn.unclaimedReward', { symbol: png.symbol })}
-                        stat={earnedAmount?.toSignificant(4)}
+                        stat={numeral(
+                          earnedAmount?.toFixed(
+                            earnedAmount?.greaterThan('0') && earnedAmount?.token?.decimals > 1 ? 2 : undefined,
+                          ),
```

Comment:
can you please add some comments here on why you did this change here ? this will help us in future to understand why we have done this.

Reply:
To fix this issue, and make the number a little smaller, and I added this condition so that it doesn't give blank screen when the earned amount is 0
https://app.clickup.com/t/8669wg4aw